standards.md

```diff
@@ -195,23 +216,30 @@ what we want.
   * Error strings are not capitalized
   * Handle all errors ([gracefully](https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully))
     * When returning errors, add more context with `fmt.Errorf` and `%v`
-  * Use meaningful package names (avoid util, helper, lib)
   * Prefer short variable names
 
+### Go packages
+
+* Prefer small well factored packages with unit tests
+* Use meaningful package names (avoid util, helper, lib)
```

Comment:
https://go.dev/blog/package-names could be a great link to include here, particularly https://go.dev/blog/package-names#bad-package-names

Reply:
Added, thanks!